src/main/java/org/littleshoot/proxy/impl/ProxyConnection.java

```diff
@@ -342,7 +399,7 @@ protected Future execute() {
 
     /**
      * Encrypts traffic on this connection with SSL/TLS.
-     * 
+     *
```

Comment:
Since this is a modification to an external project, I think it's best to only touch the parts we really changed and omit any other modification, even style-related ones by automatically applied by the IDE.

Reply:
+1